### DISCUSSION TITLE
Repar Lidl Home USB power strip HG06338

### DISCUSSION THREAD

**Discussion by MattWestb** [zigpy/zigpy#1252] [Ideas]
### First read the warning in the end !!!!
```
Device info
TS011F
by _TZ3000_1obwwnmq
Connected via [IKEA Billy EZSP]
Firmware: 0x00000045
```
I have 2 of this devices and was working good for nearly 3 years.
For 4 weeks ago one was starting doing strange things and after 2 days it was flashing the LEDs and also all relays. and it was crashing the mesh network on my production system then broadcasting parent announcement and broken routing and was need re power many routers.
Was fast looking but was not having time digging it it.

For 2 days i was here some relays playing but was not having time looking for it.
This morning the second power strip was dead.
Was looking and its one capacitor that is sitting after the switched power supply IC have growing little and one Diode in serial and the last looking going warm.
Its one TP and written 5V so it shall being the main power before the 3.3 V LIDO for the Zigbee module and its not stable then measuring it.
Now little logic: The Diode can being broken but then its shut-cut or no connection and in the last the capacitor have exploded and it have not and its some voltage so the Diode is OK.
The capacitor is 470 µF and 10 V by the book but the power supply i not very clean and the double voltage is not good for this circuit and shall being higher.
De-soldering the capacitor and putting in one 470 µF 16 V  and power the stripe on (the dead one) and its starting and the switch is working OK.
Putting it together and doing the first one and its also working OK.

So one power strip for 30 € is being broken under 3 years then have under demisonated / bad quality components for some cents.
![IMG_20230906_150305A](https://github.com/zigpy/zigpy/assets/49618193/1abedd6b-a74c-4078-883d-422b21a2bb91)
The aluminium X on top is bulged and the button is also "on the move" so the next state is exploded ones.
![IMG_20230906_150359](https://github.com/zigpy/zigpy/assets/49618193/d68e1b96-82b1-499c-84de-fed946010796)
The capacitor is not marked but is under the 5V TP marking and R3. 
U1 is the power switching IC and D2 is the Diode that is making the DC from the square wave output from it. U2 is the 3,3 V LIDO for the module.
The 16 V capacitor its little larger but is OK only need doing one pin adjustment for the legs and its working OK (Photo with the replaced capacitor).

For opening the power stripe you need first taking the silicone plugs out on the under side and then if you is having one Y screwdriver that is long enough or one 2.5 mm flat screwdriver is also working if its long enough and good quality its easy getting the 8 screw out without problems.

### Never ever have fingers, fetes, hands, children, cats, horses or lovers and other living things in the near the stripe then connecting it to the power and its open it can killing someone !!!

Now both is up and running for some hours and looks being OK.

The last one was crashing the mesh very badly and also most of the end devices must being re powered also ALL IKEA controller also the new RODRET and all router in the first level from it !!

**Comment by AlinGoga** [comment]
i hat the same problem with my power socket. i replaced that capacitor and now it's working again. After repair i found your tutorial. Thanks !

**Reply by MattWestb** [reply]
Nice to know that more have the same knowledge and also sharing it.
I was also having the same problems with one IKEA outlet and it was the same cap but they have glued the casing very hard so must putting in the oven for heating the plastic until the glue was soft and then  2 minutes for fixing it.

**Comment by nrayever-dev** [comment]
Thanks for this tutorial. My power strip suffered the same destiny.

**Comment by dorey973** [comment]
Hi, i'm having the same issue : it's blinking and I can hear the relay ticking, I don't have my tester to take mesure and there is weird things under the capacitor so I'll order the capacitor on aliexpress,
![IMG_2934](https://github.com/user-attachments/assets/2dcd13f5-d2f5-4b17-afdb-df9b024eb8fd)

**Comment by sjm42** [comment]
You know what. I fixed my HG06338 just a moment ago and took some pictures along the way.
Replaced the electrolyte with a solid one that should last basically forever and while I was at it,
replaced the hostile triangle screws with normal phillips ones.

I found this post with a search engine _after_ I had fixed it, and decided to leave a _metoo_ comment.
Greetings from Finland.

[https://photos.app.goo.gl/CBRdxwTc2KgT6H2h7](https://photos.app.goo.gl/CBRdxwTc2KgT6H2h7)

**Reply by MattWestb** [reply]
Great that you was fixing it i have done one and the second is still original but have cooking one more IKEA TF plug that also getting bad caps but they is very nasty glued and need being heated up and using much force for getting open bu then one minute fixing the cap.

**Comment by constapel** [comment]
HG06338
Thanks for the help. Now i replaced the green 470 µF 10 V with a black one found on an old PC motherboard 470µF 10 V . 
Its running again as TS011F by _TZ3000_1obwwnmq in ZHA Home Assistant.
Switching all the 3-plugs individualy.

Bought 2022-10-08
Broken  2025-09-25
Repaired 2026-01-04
Thanks for all the suggestions.

**Reply by MattWestb** [reply]
Great you was getting it fixed!!
Hope the cap from the old PC is better quality and not have drying.
The thumm is with electrolytes the working voltage *2 and never lower but with today's caps i trying putting in higher voltage if i have space for it in the device and also can going down little on the capacity if needed that normally working well and is holding longer.

**Reply by sjm42** [reply]
I would strongly suggest only using solid polymer caps, something like this one:

[https://www.digikey.com/en/products/detail/kemet/A750EQ477M1CAAE015/13420057](https://www.digikey.com/en/products/detail/kemet/A750EQ477M1CAAE015/13420057)

470µF 16V from KEMET. Of course, this is just an example of to what to look for.
This kind of capacitors should last basically "forever" in this usage case.

**Comment by kgroshert** [comment]
My powerstrip died in December 2024, fixed it by replacing the ELKO. Today it started again, a bit more than 1 year later. Will try the 16V solid polymer cap this time as @sjm42 suggested. Thanks for documenting it everyone!

**Reply by MattWestb** [reply]
THAT was FAST burnout !!!
I think its fast spikes that killing the ELKO that can handling that its made for 50/60 mains power.
If possible getting higher voltage rating and also putting one lower capacity cap parallel that is not ELKO for taking out the fast spikes.
I was putting one of my stripes on one PC and ZHA was restarting then swishing it on the PC outlet and i was finding the PC power plug was not good fitted and was doing heavy spikes and RF. Re attached the power cord and its looks working OK but you never knowing.

**Reply by sjm42** [reply]
Well, one good quality solid electrolyte cap will handle it just fine. Their internal resistance is measured in milliohms -- take a look at the link provided and check the specs on datasheet. Also, 16V voltage rating should be enough and just fine.

Cargo cult style fixing is not useful. Just put one good quality solid cap as the replacement and you'll be fine.

**Comment by MarcoFilius** [comment]
Thanks for this. Fixed mine with the same problem. scavenged a 25v 470 µF from an old tv psu and bingo!
